docs/source/user/cfunc.rst

```diff
@@ -216,3 +216,45 @@ Compilation options
 A number of keyword-only arguments can be passed to the ``@cfunc``
 decorator: ``nopython`` and ``cache``.  Their meaning is similar to those
 in the ``@jit`` decorator.
+
+
+Calling C code from Numba
+=========================
+
+It is also possible to call C code from Numba ``@jit`` functions. In this
+example, we are going to be compiling a simple function ``sum`` that adds two
+integers and calling it within Numba ``@jit`` code
+
+.. code-block:: C
+
+   #include <stdint.h>
+
+   int64_t sum(int64_t a, int64_t b){
+      return a + b;
+   }
+
+
+Compile the code with ``gcc lib.c -fpic -shared -o lib.so`` to generate a
+shared library.
+
+.. code-block:: python
+
+   from numba import njit
+   from numba.core import types, typing
+   from llvmlite import binding
+
+   # load the library into LLVM
+   binding.load_library_permanently('./path/to/lib.so')
```

Comment:
```suggestion
   binding.load_library_permanently('.//lib.so')  # path to lib.so
```

The reason I suggest this is because, when I tried this in my `/tmp` I stumbled over it. I think it would be better to be able to copy and paste the example and just have it work. I strongly presume that most uses will stuff the library next to the python file, so maybe the above would improve the user experience.

Reply:
Suggest using `os.path` utilities to handle paths.